Sources/Card/Models/Address.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+
+/// The portable international postal address.
+/// Maps to [AddressValidationMetadata](https://github.com/googlei18n/libaddressinput/wiki/AddressValidationMetadata)
+/// and HTML 5.1 [Autofilling form controls: the autocomplete attribute](https://www.w3.org/TR/html51/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute).
+public struct Address: Codable {
+
+    enum CodingKeys: String, CodingKey {
+        case addressLine1
+        case addressLine2
+        case locality = "admin_area_2"
+        case region = "admin_area_1"
+        case postalCode
+        case countryCode
+    }
+
+    /// The first line of the address. For example, number or street. For example, `173 Drury Lane`.
+    /// Required for data entry and compliance and risk checks. Must contain the full address.
+    public var addressLine1: String?
+
+    /// The second line of the address. For example, suite or apartment number.
+    public var addressLine2: String?
+
+    /// A city, town, or village. Smaller than `admin_area_level_1`.
```

Comment:
From the merchant side, they likely would know what "Smaller than `admin_area_level_1`" means since that's in our coding keys and obfuscated from them. Maybe we should update this to say smaller than region instead?

Reply:
ope I forgot to edit this!!!! :smile: